docs/DeveloperGuide.md

```diff
@@ -489,51 +496,58 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 **MSS**
 
 1.  User requests to create an internship entry with the necessary details.
-2.  InternshipBook adds the internship entry to the list.
+2.  Flagship adds the internship entry to the database.
 Use case ends.
```

Comment:
Javin, should we use the `Flagship` markdown for all references to Flagship and the `user` mark down for all references to users. I'm not too sure myself

Reply:
by markdown do u mean enclosing it in `user` , like u have done with ur comment? cause i referenced a few other UGs and they all seem to just put it as plain text. @jinyang628

Reply:
Okay got it!